src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Metrics/SocketsHttpHandlerMetrics.cs

```diff
@@ -8,25 +8,21 @@ namespace System.Net.Http.Metrics
 {
     internal sealed class SocketsHttpHandlerMetrics(Meter meter)
     {
-        public readonly UpDownCounter<long> CurrentConnections = meter.CreateUpDownCounter<long>(
-            name: "http-client-current-connections",
-            description: "Number of outbound HTTP connections that are currently active on the client.");
-
-        public readonly UpDownCounter<long> IdleConnections = meter.CreateUpDownCounter<long>(
-            name: "http-client-current-idle-connections",
-            description: "Number of outbound HTTP connections that are currently idle on the client.");
+        public readonly UpDownCounter<long> OpenConnections = meter.CreateUpDownCounter<long>(
+            name: "http.client.open_connections",
```

Comment:
The unit should be set to "{connection}"

@lmolkova - confirming that is correct and not just some formatting thing in the spec that wasn't intended to be used literally?

Reply:
Opened https://github.com/dotnet/runtime/pull/90020 PTAL.